Makefile

```diff
@@ -201,7 +201,7 @@ helm-docs: $(HELM_DOCS) ## Use helm-docs to generate documentation from helm cha
 		-t ../../../Documentation/Helm-Charts/ceph-cluster-chart.gotmpl.md \
 		-t ../../../Documentation/Helm-Charts/_templates.gotmpl
 
-check-helm-docs:
+check.helm-docs:
```

Comment:
So are going with `check.helm-docs` instead of `check.helm.docs` ?

Reply:
@travisn please take a look

Reply:
At least that is what I am still proposing 😄

Reply:
I think the general discussion about our naming conventions can be held in issue #14686

Reply:
Merging the based on above comment